provisioning/miniprov/hare_mp/utils.py

```diff
@@ -183,10 +185,53 @@ def is_component(self, machine_id: str, name: str) -> bool:
                 break
         return found
 
+    @func_log(func_enter, func_leave)
+    def is_component_services(self, machine_id: str,
+                              comp_name: str, svc_name: str) -> bool:
+        """
+        Returns True if the given component is present in the components
+        list for the given node>{machine_id} according to the
+        ConfStore (ValueProvider).
+        """
+        comp_names = self.provider.get(f'node>{machine_id}>'
```

Comment:
logically, `is_component_services()` is a combination of `is_component()` and `is_service()` both of them returning bool.
So here, we can have
```
         def is_component_services(...) -> bool:
              return self.is_component() and self.is_services()
```

Reply:
Added.